src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.component.ts

```diff
@@ -105,7 +105,10 @@ export class DatasetSettingsComponent extends BaseComponent implements OnInit {
             case SettingsTabsEnum.COMPACTION:
                 return this.datasetBasics.kind === DatasetKind.Root;
             case SettingsTabsEnum.VARIABLES_AND_SECRETS:
-                return this.appConfigService.featureFlags.enableDatasetEnvVarsManagment;
+                return (
+                    this.appConfigService.featureFlags.enableDatasetEnvVarsManagment &&
```

Comment:
It's really nice that you have a flag to disable the feature entirely - we should use this pattern more!